src/ports/postgres/modules/bayes/test/bayes.sql_in

```diff
@@ -392,5 +392,17 @@ $$ language plpgsql;
 SELECT install_test_1();
 SELECT install_test_2();
 SELECT install_test_3();
-SELECT install_test_4();
-SELECT install_test_5();
+
+-- FIXME: ORCA started to fail with GP7 beta3 while accessing bayes created views in these tests
+m4_include(`SQLCommon.m4')
+m4_changequote(`<!', `!>')
+m4_ifdef(<!__POSTGRESQL__!>, <!!>, <!
+SET optimizer = off;
+!>)
+
+ SELECT install_test_4();
```

Comment:
do these test fail with gppkg v2 installation only or with GP7 beta3 in general ?